test/page3.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Page Three
+layout: default
+---
+
+# CONTENT
+
+This page has multiple content sections under the same name.
+
+{% contentfor multi %}
+---
+number: 1
+---
+  The first section
+{% endcontentfor %}
+
+{% contentfor multi %}
+---
+number: 2
+---
+  The second section
+{% endcontentfor %}
+
+{% contentfor multi %}
+---
+number: 3
+---
+  The third section
+{% endcontentfor %}
+
```

Comment:
This is great but there's one more step -- in the spec file, could you please add some assertions in `jekyll_content_blocks_spec` to make sure this gets rendered correctly? Perhaps in a new `describe 'page3.html'` section?